plugin/src/main/java/io/jenkins/plugins/casc/SchemaObjects.java

```diff
@@ -0,0 +1,9 @@
+package io.jenkins.plugins.casc;
+
+import javax.validation.constraints.NotNull;
+
+public class SchemaObjects {
+
+    @NotNull
+    public String foo;
```

Comment:
This is not the best way of attempting to construct the schema but , we need to figure out a way to get the major plugins supported via something like this.Nesting needs to covered as well.How many plugins in total does JCasC support ?